### DISCUSSION TITLE
MacOS: Is there a way to have multiple windows?

### DISCUSSION THREAD

**Discussion by jlbprof** [Eugeny/tabby#4652] [Q&A] [answered]
I love the tabs, but I often need multiple windows.   I could not figure out how to open another window.

Thank you

Bodger

**Comment by Eugeny** [comment, marked as answer]
Cmd-N (Settings -> Hotkeys -> New window)

**Reply by jlbprof** [reply]
Yes that worked thank you

**Comment by lawweiliang** [comment]
@jlbprof   
Please close the discussion if the question was answered. 
It would be helpful to make the discussion section clean. 

Click `mark as answered` and `close the discussion`. 😄 

![image](https://user-images.githubusercontent.com/34936929/228098333-222ac3d4-4c39-4c26-a2a8-52906647e00e.png)
